src/Models/ElementContent.php

```diff
@@ -41,6 +41,13 @@ public function getSummary()
         return DBField::create_field('HTMLText', $this->HTML)->Summary(20);
     }
 
+    public function getSummaryData()
+    {
+        $summaryData = [];
```

Comment:
See previous comment in BaseElement about this. Whichever method is used, this particular line should be `$summaryData = parent::getSummaryData();` rather than entirely overloading it